compiler/wasm/src/types/noir_artifact.ts

```diff
@@ -145,14 +145,22 @@ export type BrilligFunctionId = number;
 
 export type OpcodeToLocationsMap = Record<OpcodeLocation, SourceCodeLocation[]>;
```

Comment:
`OpcodeLocation` can be serialized to a number now that it's just the ACIR index.